<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="subjBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Физика</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Алгебра</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Геометрия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Русский язык</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Литература</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Английский язык</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Обществознание</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>История</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ИП</string>
     </property>
    </item>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dirButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Dir</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="nameText">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="classText">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
